### PR TITLE
Fixing observing/pointing wait events

### PR DESCRIPTION
**Pull request by wtgee**
Removing units from the MAX_EXTRA_TIME for consistency.

This is fixing a bug introduced in #625.

@jamessynge note that this is fixing those problems with testing the flaws in our testing I was talking about. The bug was causing a shutdown in the pointing state but most of the tests in `test_pocs` report as passing because POCS itself was absorbing the errors.

### CODE REVIEW

File: pocs/tests/utils/test_utils.py [hunk 1]
```diff
@@ -51,3 +52,30 @@ def test_has_camera_ports():
...
+    assert timer.is_non_blocking
+    assert timer.time_left() == 0
+
+
+def test_countdown_timer():
+    timer = CountdownTimer(1)
+    assert timer.time_left() > 0
+    assert timer.expired() is False
+    assert timer.is_non_blocking is False
+    time.sleep(1)
```
Comment (jamessynge) [inline pocs/tests/utils/test_utils.py]: I'd prefer a sleep time that is greater than the count down time. Alternately, you could have a loop that checks every 0.01 seconds (for example), and tracks whether the time_left is decreasing each time, and goes to zero before 1.1 seconds (for example) has passed.

File: pocs/tests/utils/test_utils.py [hunk 2]
```diff
@@ -51,3 +52,30 @@ def test_has_camera_ports():
...
+        assert CountdownTimer('d')
+
+    with pytest.raises(ValueError):
+        assert CountdownTimer(current_time())
+
+    with pytest.raises(AssertionError):
+        assert CountdownTimer(-1)
+
+
+def test_countdown_timer_non_blocking():
```
Comment (jamessynge) [inline pocs/tests/utils/test_utils.py]: Could you add tests with float and u.second? For example:
```
for arg, expected_duration in [(2, 2.0), (0.5, 0.5), (1 * u.second, 1.0)]:
    timer = CountdownTimer(arg)
    assert timer.duration == expected_duration
...
```

File: pocs/tests/utils/test_utils.py (hunk 1 above)
Comment (wtgee) [reply pocs/tests/utils/test_utils.py]: Good idea. I update this, let me know if you think still needs improvement.

File: pocs/tests/utils/test_utils.py (hunk 2 above)
Comment (wtgee) [reply pocs/tests/utils/test_utils.py]: Done, thanks.

**Pull request merged**